db/migrate/0004_add_position_to_issue_templates.rb

```diff
@@ -3,8 +3,11 @@ def self.up
     add_column :issue_templates, :position, :integer, default: 1
 
     IssueTemplate.reset_column_information
+
     issue_templates = IssueTemplate.all
-    issue_templates.each_with_index { |t, i| t.update_attribute(:position, i + 1) }
+    say_with_time("Update each template to set default position.") do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)